stdlib/grpc/build.gradle

```diff
@@ -49,6 +49,7 @@ dependencies {
     interopImports project(':ballerina-io')
 
     implementation 'org.apache.commons:commons-lang3'
+    implementation 'com.google.api.grpc:proto-google-common-protos:1.17.0'
```

Comment:
Do we need the specific version here?

Reply:
The documentation for the external resource suggested including the dependency in such a manner in gradle.

Reply:
Shall we add this dependency with the version to here[1] and refer it here without the version. So we are maintaining the version in one place

1. https://github.com/ballerina-platform/ballerina-lang/blob/master/gradle/javaLibsProject.gradle.

Reply:
Removed the version and added with version to [1] described above